_data/jobs.yml

```diff
@@ -1,36 +1,34 @@
-- {expires: 2020-12-01, location: 'Harvard University, Cambridge, MA',
+- {expires: 2021-03-01, location: 'Harvard University, Cambridge, MA',
     name: Data Scientist / Engineer,
     url: 'https://sjobs.brassring.com/TGnewUI/Search/home/HomeWithPreLoad?partnerid=25240&siteid=5341&PageType=JobDetails&jobid=1542050#jobDetails=1542050_5341'}
-- {expires: 2020-12-01, location: 'Harvard University, Cambridge, MA',
+- {expires: 2021-03-01, location: 'Harvard University, Cambridge, MA',
     name: Research Software Engineer (Scientific Software Developer),
     url: 'https://www.rc.fas.harvard.edu/about/employment/#Research_Software_Engineer_Scientific_Software_Developer'}
-- {expires: 2020-12-01, location: 'Amazon Web Services, Inc., Seattle, WA (potential for remote)',
+- {expires: 2020-03-01, location: 'Amazon Web Services, Inc., Seattle, WA (potential for remote)',
```

Comment:
```suggestion
- {expires: 2021-03-01, location: 'Amazon Web Services, Inc., Seattle, WA (potential for remote)',
```

Reply:
Is there a certain number of months to choose for each or does it vary? I thought I saw a pattern but I’m not sure now 🤔